src/parser/shared/modules/CooldownThroughputTracker.tsx

```diff
@@ -25,14 +25,14 @@ type TrackedEvent = CastEvent | HealEvent | AbsorbedEvent | DamageEvent | ApplyB
 
 export type CooldownSpell = {
   spell: any,
-  summary: Array<BUILT_IN_SUMMARY_TYPES>,
+  summary: Array<BUILT_IN_SUMMARY_TYPES | any>,
```

Comment:
This is blocking, please remove this and we can work to fix whatever issue it was added to resolve

Reply:
Yeah, I didn't really know how to handle this one. I have no idea what the module I changed this for was trying to accomplish.

https://github.com/WoWAnalyzer/WoWAnalyzer/blob/93f86fead27addf6aec9a2d5860f6e43f246bb45/src/parser/priest/discipline/modules/features/CooldownThroughputTracker.ts#L43

I was trying to fix the typing here ^^ 

I'd LOVE some additional insight into what this is actually doing.

Reply:
For now I've commented out the code in that module

Reply:
I had to look into this,

![image](https://user-images.githubusercontent.com/43309639/95018092-3ac37c80-0612-11eb-9dc2-5207fac0753d.png)

it looks like its perfectly valid, and used in interface/others/Cooldown.js L 294-297

In the core CooldownThroughputTracker (i.e. the file this comment was originally for):
```typescript
export type SummaryDef = {
  label: string,
  tooltip: string,
  value: number | string,
}
export type CooldownSpell = {
  spell: any,
  summary: Array<BUILT_IN_SUMMARY_TYPES | SummaryDef>,
  startBufferFilter?: EventFilter<any>,
  startBufferMS?: number,
  startBufferEvents?: number,
  petID?: number,
};
```